config/locales/sk.yml

```diff
@@ -286,6 +286,7 @@ sk:
   "Automation::MetadataValueNotCondition": "v metadátach neobsahuje"
   "Automation::MessageMetadataValueCondition": "správa v metadátach obsahuje"
   "Automation::BoxCondition": "je"
+  "Automation::APIConnectionCondition": "je"
```

Comment:
Pridaj este hore preklad nech sa v select boxe ukazuje API Prepojenie ako to je v menu prelozene